.gitignore

```diff
@@ -8,3 +8,4 @@ pep8.txt
 dist/
 build/
 .tox
+venv/
```

Comment:
Why?

Reply:
I didn't find any environment directory name and added `venv`.